static/js/apps/eval_rig/feedback_form.tsx

```diff
@@ -14,181 +14,165 @@
-            <input
-              type="radio"
-              name="question"
-              value="1"
-              checked={responses.question === "1"}
-              onChange={handleChange}
-            />
-            Irrelevant
-          </label>
-
-          <h3>Model response quality</h3>
-          <label>
-            <input
-              type="radio"
-              name="llmResponse"
-              value="0"
-              checked={responses.llmResponse === "0"}
-              onChange={handleChange}
-            />
-            Stats Seem Accurate
-          </label>
-          <label>
-            <input
-              type="radio"
-              name="llmResponse"
-              value="1"
-              checked={responses.llmResponse === "1"}
-              onChange={handleChange}
-            />
-            Stat values seem off
-          </label>
+          <OneQuestion
+            question="Question from the model"
+            name="question"
+            values={{
+              DC_QUESTION_RELEVANT: "Well formulated & relevant",
+              DC_QUESTION_IRRELEVANT: "Irrelevant, vague, requires editing",
+            }}
+            handleChange={handleChange}
+            responseField={response.question}
+          />
+          <OneQuestion
+            question="Model response quality"
+            name="llmResponse"
+            values={{
+              LLM_STAT_ACCURATE: "Stats seem accurate",
+              LLM_STAT_INACCURATE: "Stats seem accurate",
+            }}
+            handleChange={handleChange}
+            responseField={response.llmResponse}
+          />
         </div>
 
         <div>
           <h2>DATA COMMONS EVALUATION</h2>
           <h3>{props.evalInfo.dcResponse}</h3>
           <h3>{props.evalInfo.dcStat}</h3>
-          <h3>Response from Data Commons</h3>
-          <label>
-            <input
-              type="radio"
-              name="dcResponse"
-              value="0"
-              checked={responses.dcResponse === "0"}
-              onChange={handleChange}
-            />
-            Relevant and direct
-          </label>
-          <label>
-            <input
-              type="radio"
-              name="dcResponse"
-              value="1"
-              checked={responses.dcResponse === "1"}
-              onChange={handleChange}
-            />
-            Doesn&apos;t match the question
-          </label>
+          <OneQuestion
+            question="Response from Data Commons"
+            name="dcResponse"
+            values={{
+              DC_ANSWER_RELEVANT: "Relevant and direct",
+              DC_ANSWER_IRRELEVANT: "Doesn't match the question",
+              DC_ANSWER_EMPTY_BADNL: "Data exists, but NL fails to respond",
+              DC_ANSWER_EMPTY_NODATA:
+                "Query asks for data that doesn't exist in DC",
+              DC_ANSWER_EMPTY_OUTOFSCOPE:
+                "Query asks for data that is out-of-scope for DC",
+            }}
```

Comment:
I think the DC_ANSWER_EMPTY_* ones are relevant only when the DC answer is empty.

Likewise, when DC answer is empty, DC_ANSWER_RELEVANT/IRRELEVANT don't make sense.

Reply:
updated